[DOC-2499] & related - CI build infrastructure refresh + more Hosted content (#898)
harness/developer-hub: docs/continuous-integration/use-ci/set-up-build-infrastructure/vm-build-infrastructure/define-a-ci-build-infrastructure-in-google-cloud-platform.md

Next, you'll add the Runner spec to the new Delegate definition. The Harness Delegate and Runner run on the same VM. The Runner communicates with the Harness Delegate on `localhost` and port `3000` of your VM.

> 6. Copy your local `docker-compose.yaml` file to the `/runner` folder on the VM. This folder should now have both `docker-compose.yaml` and `pool.yml`.

Reviewer: ```suggestion
1. Copy your local `docker-compose.yaml` file to the `/runner` folder on the VM. This folder should now have both `docker-compose.yaml` and `pool.yml`.
```

> 7. Open `docker-compose.yaml` in a text editor.

Reviewer: ```suggestion
2. Open `docker-compose.yaml` in a text editor.
```

> 8. Append the following to the end of the `docker-compose.yaml` file:

Reviewer: ```suggestion
3. Append the following to the end of the `docker-compose.yaml` file:
```

Reviewer: Also renumber remaining steps in this procedure...

> 

Reviewer: Revise step 4 as follows:
Verify that both containers are running correctly. You might need to wait a few minutes for both processes to start. You can run the following commands to check the process status:

The Delegate and Runner are now installed, registered, and connected.

## Step 5: Select pipeline build infrastructure

1. In your CI pipeline's **Build** stage, select the **Infrastructure** tab, and then select **VMs**.
2. In the **Pool ID**, enter the pool `name` from your [pool.yml](#step-2-configure-the-drone-pool-on-the-google-vm).

   ![](../static/define-a-ci-build-infrastructure-in-google-cloud-platform-31.png)

This pipeline's **Build** stage now uses your GCP VMs for its build infrastructure.

## Pool Settings Reference

> You can configure the following settings in your pool.yml file.

Reviewer: ```suggestion
You can configure the following settings in your `pool.yml` file.
```

   ```
4. Restart the delegate and runner: `docker-compose up`

## See Also

* [Set up a Kubernetes cluster build infrastructure](../set-up-a-kubernetes-cluster-build-infrastructure.md)

> * For more details on VM Runner, visit this [GitHub](https://github.com/drone-runners/drone-runner-aws) page.

Reviewer: ```suggestion
* For more details on VM Runner, go to this [GitHub](https://github.com/drone-runners/drone-runner-aws) page.
```
